Tests/SDK/Service/HttpRequestRunnerTest.swift

```diff
@@ -0,0 +1,45 @@
+@testable import CIO
+import Foundation
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
+import XCTest
+
+/**
+ Test that performs a real HTTP request to Customer.io API. This is a convenient way to test the
+ networking layer of the SDK. These tests are meant to run on your local machine, not CI server.
+
+ Setup:
+ 1. In XCode, go to: Edit Scheme > Run
+ 2. Create 2 environment variables: `SITE_ID` and `API_KEY`. Populate those values with random credentials from a Workspace.
+ 3. Manually run the tests below. Use the XCode debug console to see the log output for debugging.
+ */
+class HttpRequestRunnerTest: UnitTest {
```

Comment:
so is this going to be run by our CI as well?

Reply:
As the code stands here, no. The code is setup to only run these tests if the 2 environment variables are present else, skip the tests. 

I did try isolating the iOS networking code to a very small class `HttpRequestRunner`. I believe that as long as that class's code does not get edited we can assume it will work because it's just performing an OS API call. It's just a wrapper for testing purposes. The class that does most of the networking handling `HttpClient` is tested well with mocking. 

With that being said, I don't see a need to enable it on the CI server. You have a scenario in mind it makes sense?

Reply:
nope, other way around. seemed like the kind of test to make sure CI ignored